daemon/containerd/image_delete_test.go

```diff
@@ -0,0 +1,289 @@
+package containerd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/metadata"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/log/logtest"
+	"github.com/docker/docker/container"
+	daemonevents "github.com/docker/docker/daemon/events"
+	dockerimages "github.com/docker/docker/daemon/images"
```

Comment:
Let's keep the aliases consistent:
```suggestion
	dimages "github.com/docker/docker/daemon/images"
```